frontend/src/app/shared/components/work-package-graphs/embedded/wp-embedded-graph.component.ts

```diff
@@ -91,9 +91,7 @@ export class WorkPackageEmbeddedGraphComponent {
         label: dataset.label,
         data: uniqLabels.map((label) => countMap[label] || 0),
         borderColor,
-        backgroundColor: this.chartType === 'bar' || this.chartType === 'horizontalBar'
-          ? uniqLabels.map((_, i) => this.getHexColor(i))
-          : undefined,
+        backgroundColor: uniqLabels.map((_, i) => this.getHexColor(i))
```

Comment:
Setting `backgroundColor: undefined` appears to behave differently with the upgraded version of ng2-charts.

I think applying `backgroundColor` to all chart types is probably what we want.